### DISCUSSION TITLE
Duvidas de iniciante

### DISCUSSION THREAD

**Discussion by DanielRodrigues23** [balta-io/forum#322] [Tira Dúvidas] [answered]
Sou iniciante em programação, e estou terminando fundamentos do c#, gostaria de saber se esses projetos de aprendizagem do curso como, calculadora, cronometro e etc, seriam bom ja para publicarem no git de um iniciante, mesmo nao sendo algo avançado ?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @Dsr1211, como vai!

Com certeza... Se você olhar meu GitHub vai ver que tudo que estudo publico lá, desde Hello World até coisas mais complexas.
Um bom portifólio demostra que você tem empenho, que se mantém atualizado.

🚀
